src/main/java/org/kuali/kra/protocol/onlinereview/ProtocolOnlineReviewLookupableHelperServiceImplBase.java

```diff
@@ -222,4 +223,27 @@ private AnchorHtmlData getAnchorHtmlData(Properties parameters, String displayTe
         String href  = UrlFactory.parameterizeUrl("../" + getHtmlAction(), parameters);
         return new AnchorHtmlData(href, KRADConstants.DOC_HANDLER_METHOD, displayText);
     }
+
+    @Override
+    public void performClear(LookupForm lookupForm) {
+        for (Iterator iter = this.getRows().iterator(); iter.hasNext();) {
+            Row row = (Row) iter.next();
+            for (Iterator iterator = row.getFields().iterator(); iterator.hasNext();) {
+                Field field = (Field) iterator.next();
+                if (field.isSecure()) {
+                    field.setSecure(false);
+                    field.setDisplayMaskValue(null);
+                    field.setEncryptedValue(null);
+                }
+
```

Comment:
Jim, please add a comment here above excluding the field with a very short explanation why we're excluding it

Reply:
Natalia, I added a comment explaining why we are Not clearing 'lookupReviewerPersonId' so that we still search for the logged in reviewer even after a search parameter clear.

Reply:
Looks great!